### PR TITLE
run warmup on one node and multinode after

### PR DESCRIPTION
**Pull request by twicki**
## Purpose

- This PR changes from using the pre-installed vcm_venv on daint to installing it's own for performance runs.
- To get fair performance numbers, we do a compilation run first on one node and move to a multi-node execution with the already compiled code afterwards

## Code changes:

- only changes how run_on_daint is executing.

### CODE REVIEW

File: examples/standalone/benchmarks/run_on_daint.sh [hunk 1]
```diff
@@ -83,19 +88,39 @@ else
...
+wait
+echo "compilation step finished"
+rm *.json
+
+echo "submitting script to do performance run"
+# Adapt batch script to run the code:
+sed -i s/\<NAME\>/standalone/g run.daint.slurm
+sed -i s/\<NTASKS\>/$ranks/g run.daint.slurm
+sed -i s/\<NTASKSPERNODE\>/1/g run.daint.slurm
+sed -i s/\<CPUSPERTASK\>/12/g run.daint.slurm
```
Comment (eddie-c-davis) [inline examples/standalone/benchmarks/run_on_daint.sh:115]: Not a big deal, but should this be using $nthreads?

File: examples/standalone/benchmarks/run_on_daint.sh [hunk 2]
```diff
@@ -83,19 +88,39 @@ else
...
+sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g run.daint.slurm
+sed -i s/00:45:00/00:30:00/g run.daint.slurm
+sed -i s/cscsci/debug/g run.daint.slurm
```
Comment (eddie-c-davis) [inline examples/standalone/benchmarks/run_on_daint.sh:118]: Does the `debug` queue work with multiple nodes?

**Review by eddie-c-davis — APPROVED** [review]
Looks good, but have a couple small questions...

File: examples/standalone/benchmarks/run_on_daint.sh (hunk 1 above)
Comment (twicki) [reply examples/standalone/benchmarks/run_on_daint.sh:115]: yeah - oversight. Changed

File: examples/standalone/benchmarks/run_on_daint.sh (hunk 2 above)
Comment (twicki) [reply examples/standalone/benchmarks/run_on_daint.sh]: it does work. But I just compared with our old data where a timestep is still close to 45' with the c128 data, i switched it to normal.

**Review by eddie-c-davis — APPROVED** [review]
Looks great!

**Pull request merged**